### PR TITLE
Added doc for console access.

### PR DESCRIPTION
**Pull request by tomspilman**
@SimonDarksideJ @harry-cpp @mrhelmut any input on this?

### CODE REVIEW

**Comment by tomspilman** [comment]
For reference of what it looks like:

image

image

**Review by SimonDarksideJ — APPROVED** [review]

**Comment by mrhelmut** [comment]
Looks good to me!

File: articles/console_access.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Console Access
+description: How do I get access to console versions of MonoGame?
+---
+
+MonoGame is free to use on all consoles from the [public repository](https://github.dev/MonoGame/monogame), however, the source code to the extensions for each platform is private and only accessible to authorized developers for each platform (managed by the vendors).  These extensions provide additional capabilities to enable MonoGame to perform better on those platforms and include integrations with the vendor's private APIs.
```
Comment (tomspilman) [inline articles/console_access.md:6]: @SimonDarksideJ

Can you remove "to perform better on those platforms"?  That is just not true.

The purpose of this private code is strictly to make MonoGame work on those platforms.  It is not performance enhancements.

Comment (tomspilman) [reply articles/console_access.md:6]: Actually i'll rewrite this really quick and submit a new PR.

**Pull request merged**